tree_unique_args/src/schedule.egg

```diff
@@ -3,6 +3,7 @@
 (run-schedule
   (repeat 5
     (saturate always-run)
+    (saturate error-checking) ; In the future, this will be "release mode" only
```

Comment:
did you mean "debug mode" only?
I think it's fine to run these in release unless we think they are slow

Reply:
Yes, pretty sure this should be debug mode only (if other error-checking rules end up slow)